addons/website/data/website_data.xml

```diff
@@ -601,21 +601,21 @@
             <field name="type">url</field>
             <field name="url">/website/static/src/img/snippets_demo/s_text_image.jpg</field>
         </record>
-        <record id="website.s_three_columns_image_default_1" model="ir.attachment">
+        <record id="website.s_three_columns_default_image_1" model="ir.attachment">
             <field name="public" eval="True"/>
-            <field name="name">s_three_columns_image_default_1.jpg</field>
+            <field name="name">s_three_columns_default_image_1.jpg</field>
             <field name="type">url</field>
             <field name="url">/web/image/website.library_image_11</field>
         </record>
-        <record id="website.s_three_columns_image_default_2" model="ir.attachment">
+        <record id="website.s_three_columns_default_image_2" model="ir.attachment">
             <field name="public" eval="True"/>
-            <field name="name">s_three_columns_image_default_2.jpg</field>
+            <field name="name">s_three_columns_default_image_2.jpg</field>
             <field name="type">url</field>
             <field name="url">/web/image/website.library_image_13</field>
         </record>
-        <record id="website.s_three_columns_image_default_3" model="ir.attachment">
+        <record id="website.s_three_columns_default_image_3" model="ir.attachment">
             <field name="public" eval="True"/>
-            <field name="name">s_three_columns_image_default_3.jpg</field>
+            <field name="name">s_three_columns_default_image_3.jpg</field>
```

Comment:
@Cocographique Oh ok, you renamed to the right name you used in the other commit here ^^"
Anyways, I am not sure this is a good idea to rename those, they are used by people in their template, so you break all website using those for a convention reason :/ I'll leave the commit out my review, you can move it to another PR to be judged later.

Reply:
**PR** [46829](https://github.com/odoo/odoo/pull/46873)